static/styles/left_sidebar.css

```diff
@@ -190,22 +188,92 @@ li.show-more-topics {
     }
 }
 
-#private-container,
-#stream-filters-container {
+#pm_stream_scroll_container {
+    outline: none;
     overflow-x: hidden;
     overflow-y: auto;
     position: relative;
     z-index: 0;
     width: 100%;
 }
 
-#private-container {
-    max-height: 210px;
+#private_messages_section {
+    margin-bottom: $sections_vertical_gutter;
 
-    /* Match the opacity for global-filters icons. */
-    span.fa-group {
+    .fa-caret-down,
+    .fa-chevron-left {
         opacity: 0.7;
     }
+    
+    #private_messages_section_header {
```

Comment:
I moved all the relevant css to left sidebar PMs here so that they are easier to maintain and css of properties and be overridden without conflict.